src/tactic/ui/panel/collection_wdg.py

```diff
@@ -0,0 +1,852 @@
+
+        my.kwargs["scale"] = 75;
+        my.kwargs["show_scale"] = False;
+
+
+        from tile_layout_wdg import TileLayoutWdg
+        tile = TileLayoutWdg(
+            **my.kwargs
+        )
+
+        path = my.kwargs.get("path")
+        if collection and path:
+            title_div = DivWdg()
+            top.add(title_div)
+            title_div.add_style("float: left")
+            title_div.add_style("margin: 15px 0px")
+
+            path = path.strip("/")
+            parts = path.split("/")
+
+            for part in parts:
+                title_div.add(" / ")
+                title_div.add(" <a class='spt_colleciton_link'><b>%s</b></a> " % part)
+                title_div.add_style("margin-top: 10px")
+                title_div.add_style("margin-left: 20px")
+            #title_div.add("/ %s" % collection.get_value("name") )
+
+        #scale_wdg = tile.get_scale_wdg()
+        #top.add(scale_wdg)
+        #scale_wdg.add_style("float: right")
+
+        top.add(my.get_header_wdg())
+
+        top.add(tile)
+
+        return top
+
+ 
+
+    def get_header_wdg(my):
+
+        div = DivWdg()
+
+        if my.collection_key:
+
+            button = IconButtonWdg(title='Remove Selected Items from Collection', icon="BS_MINUS")
+            div.add(button)
+            button.add_style("display: inline-block")
+            button.add_style("vertical-align: top")
+
+            button.add_behavior( {
+                'type': 'click_up',
+                'collection_key': my.collection_key,
+                'cbjs_action': '''
+                var search_keys = spt.table.get_selected_search_keys(false);
+
+                if (search_keys.length == 0) {
+                    spt.notify.show_message("Nothing selected to remove");
+                    return;
+                }
+
+                var cls = 'tactic.ui.panel.CollectionRemoveCmd';
+                var kwargs = {
+                    collection_key: bvr.collection_key,
+                    search_keys: search_keys,
+                }
+                var server = TacticServerStub.get();
+                server.execute_cmd(cls, kwargs);
+
+                spt.table.remove_selected();
+
+                '''
+            } )
+
+
+
+            button = IconButtonWdg(title='Delete Collection', icon="BS_TRASH")
+            #button = ActionButtonWdg(title='Delete Collection', icon="BS_TRASH")
+            div.add(button)
+            button.add_style("display: inline-block")
+            button.add_style("vertical-align: top")
+
+            button.add_behavior( {
+                'type': 'click_up',
+                'collection_key': my.collection_key,
+                'cbjs_action': '''
+
+                var cls = 'tactic.ui.panel.CollectionDeleteCmd';
+                var kwargs = {
+                    collection_key: bvr.collection_key,
+                }
+                var server = TacticServerStub.get();
+                server.execute_cmd(cls, kwargs);
```

Comment:
please wrap execute_cmd  ...... till run_search() in try { } catch(e) { spt.alert(spt.exception.handler(e));}